integration/conftest.py

```diff
@@ -312,68 +312,19 @@ def bob(reactor, temp_dir, introducer_furl, flog_gatherer, storage_nodes, reques
 def chutney(reactor, temp_dir: str) -> tuple[str, dict[str, str]]:
     """
     Install the Chutney software that is required to run a small local Tor grid.
-
-    (Chutney lacks the normal "python stuff" so we can't just declare
-    it in Tox or similar dependencies)
     """
-    # Try to find Chutney already installed in the environment.
-    try:
-        import chutney
-    except ImportError:
-        # Nope, we'll get our own in a moment.
-        pass
-    else:
-        # We already have one, just use it.
-        return (
-            # from `checkout/lib/chutney/__init__.py` we want to get back to
-            # `checkout` because that's the parent of the directory with all
-            # of the network definitions.  So, great-grand-parent.
-            FilePath(chutney.__file__).parent().parent().parent().path,
-            # There's nothing to add to the environment.
-            {},
-        )
-
-    chutney_dir = join(temp_dir, 'chutney')
-    mkdir(chutney_dir)
+    import chutney
 
     missing = [exe for exe in ["tor", "tor-gencert"] if not which(exe)]
     if missing:
         pytest.skip(f"Some command-line tools not found: {missing}")
 
-    # XXX yuck! should add a setup.py to chutney so we can at least
-    # "pip install <path to tarball>" and/or depend on chutney in "pip
-    # install -e .[dev]" (i.e. in the 'dev' extra)
-    #
-    # https://trac.torproject.org/projects/tor/ticket/20343
-    proto = _DumpOutputProtocol(None)
-    reactor.spawnProcess(
-        proto,
-        'git',
-        (
-            'git', 'clone',
-            'https://gitlab.torproject.org/tpo/core/chutney.git',
-            chutney_dir,
-        ),
-        env=environ,
+    return (
+        # The directory with all of the network definitions.
+        FilePath(chutney.__file__).parent().child("data").path,
+        # There's nothing to add to the environment.
+        {},
```

Comment:
Should be able to just make the `chutney` fixture return the data path (only) then right? (That is, probably the return-type should be ``-> FilePath`` alone not a tuple, and not a str)

Reply:
Yes!  Thanks.  Will do.

But I'll try to first make it work again before doing refactors.  Seems TOR integration tests haven't been running for quite a while.